.gitignore

```diff
@@ -0,0 +1,17 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+
+# Ignore the default SQLite database.
+/db/*.sqlite3
+/db/*.sqlite3-journal
+
+# Ignore all logfiles and tempfiles.
+/log/*
+!/log/.keep
+/tmp
```

Comment:
Missed values for ignore, there are usually 3 groups of files to ignore:
1. IDE-specific settings (if team not share same settings)
2. Secrets (API keys etc ....)
3. Logs and other system temp files, such as cache.

For example, you can add:
.idea
# Secret files
.env.local
public/javascripts
.byebug_history

etc.

And, specific for this project:
config/secrets.yml